languages/go/reference/exercise-concepts/parallel-letter-frequency.md

```diff
@@ -0,0 +1,29 @@
+# Concepts of Parallel Letter Frequency
+
+## Concurrency
+
+The goal of this exercise is to learn to write concurrent code to count the letters inside different texts.
+
+## Goroutines
+
+In order to count concurrent the apppearance of different letters in separate texts it is important to be able to create goroutines with the `go` keyword, one for each text where the letters are counted.
```

Comment:
The goroutine should also do something useful! I've seen many solutions where the goroutine simply iterates the letters of the input string and feeds them to the channel. Not, in the scheme of things, all that helpful.

Reply:
@bitfield: which concept do you think is missing?